### ISSUE TITLE
PRS IDs and LOC IDs for Mikāʾel from Atripe and Yoḥannəs from Burləs

### ISSUE THREAD

**Issue by CarstenHoffmannMarburg** [BetaMasaheft/Documentation#2364]
@thea-m @DenisNosnitsin1970 @eu-genia 

I want to encode the introductory text of a synaxary in BMLacq680. I don't know yet, if it is the first or the second recension, because the incipits seem to me identical.

I want to encode also the two persons, that the work is attributed to:
አባ፡ ሚካኤል፡ ኤጲስ፡ ቆጶስ፡ ዘሀገረ፡ አትሪብ፡ ወመሊግ
and: አባ፡ ዮሐንስ፡ ኤጲስቆጶስ፡ ዘሀገረ፡ ቡርልስ
I am not sure, if the former is recorded yet. The second may be PRS11934Yohannes, because there is a relation to the Synaxary, but I am also not sure about it, because I don't understand, what kind of relation it is.
Shall I create new PRS ID for them or do they exist already?
Can I create also LOC IDs for the place names? How can they be identified?
አትሪብ = Atripe in Lower Egypt (not the one in Upper Egypt)
መሊግ = ?
ቡርልስ = ?

**Comment by thea-m** [comment]
(It's the second recension - this can be noted relatively quickly by the presence of salāms)
Michael has the ID PRS7054Michael, I think there is none for Yoḥannəs of Burləs (PRS11934Yohannes refers to a saint commemorated on 8 Maskaram). You can create LOC IDs for place names (referring to Solomon's and Pietro's conceptual work that is linked to in the places guidelines). 
Again, the caveat - starting to record all names that are attested only as attribute of protagonists will be an endless task. If you have a clear aim with creating some of them, there is no problem, but it might be better to spend more time on manuscript description.

**Comment by CarstenHoffmannMarburg** [comment]
I encoded Atripe. I would like to mention also the Coptic name ⲁⲑⲣⲏⲃ, but XML did not display the coptic letters, so I obstained.

**Comment by CarstenHoffmannMarburg** [comment]
I also want to mention, that the White monastery is not far from it, but I don't know, how to encode that as a relation.

**Comment by CarstenHoffmannMarburg** [comment]
For ቡርልስ I found Burullus in Lower Egypt. There has been a town with this name in ancient times, but nowadays, there is just a lake, that carries this name. I want to encode it, because it is difficult to identify.

https://en.wikipedia.org/wiki/Lake_Burullus

**Comment by CarstenHoffmannMarburg** [comment]
What about Johannes? If there is no ID for him yet, I want to create a new one.

**Comment by thea-m** [comment]
I think there is none, so please go ahead!

**Comment by eu-genia** [comment]
Note that from 

> I encoded Atripe. I would like to mention also the Coptic name ⲁⲑⲣⲏⲃ, but XML did not display the coptic letters, so I obstained.

you can use Coptic but as long as you use Titus as the main font in Oxygen it won't be visible in the editor as Titus does not have a Coptic subset. Unfortunately there is no single font for all scripts. If you use Ludolfus you would see Coptic and Ethiopic but you would not see Arabic, if you use Serif you would see Arabic and Coptic but not Ethiopic etc etc. You still can insert it and it will be visible online.

**Comment by eu-genia** [comment]
> I also want to mention, that the White monastery is not far from it, but I don't know, how to encode that as a relation.

you can say
`<relation name="dcterms:relation" active="xxxx" passive="xxx"><desc>Any description of the relation (e.g. that it is located close to the WM)</desc></relation>`

**Comment by eu-genia** [comment]
Note also that for Coptic tradition it would be very good to align our data to the PATHS database https://atlas.paths-erc.eu/ which has authority files for persons, places, works, etc. and links to the Trismegistos and Pleiades authority files

Paralos is here: https://atlas.paths-erc.eu/places/132

**Comment by eu-genia** [comment]
Malij: https://atlas.paths-erc.eu/places/383

**Comment by eu-genia** [comment]
Atripe (monastery): https://atlas.paths-erc.eu/places/112

**Comment by CarstenHoffmannMarburg** [comment]
How can I encode IDs from Trismegistos and Plejades?

**Comment by CarstenHoffmannMarburg** [comment]
> Michael: https://ccdl.claremont.edu/digital/collection/cce/id/1345/rec/1464

This source states that Michael is from Atripe in Lower Egypt, but not from Atripe in Upper Egypt, that I have just encoded. I will create one more ID for Atripe in Lower Egypt and correct the relation.

**Comment by eu-genia** [comment]
> How can I encode IDs from Trismegistos and Plejades?

Paths is enough

**Comment by CarstenHoffmannMarburg** [comment]
> Atripe (monastery): https://atlas.paths-erc.eu/places/112

This record points on the monastery, but not to the town in the strict sense. I will use it for the record of LOC2594Dayral than. Is it correct to use the Paths-ID (paths.places.112) in connection with sameAs= in body -> listPlace -> place   ?

**Comment by CarstenHoffmannMarburg** [comment]
> > Michael: https://ccdl.claremont.edu/digital/collection/cce/id/1345/rec/1464
> 
> This source states that Michael is from Atripe in Lower Egypt, but not from Atripe in Upper Egypt, that I have just encoded. I will create one more ID for Atripe in Lower Egypt and correct the relation.

I think, I was mistaken in stating, that Michael of Atripe is from Atripe in Lower Egypt. In fact it is the Synaxary from Lower Egypt. But Michael is referred to be the bishop of Atripe and Malij. Since Malij is in Upper Egypt not far from Atripe near Suhaj, I need to rework the record again and I need to relate Michael to LOC7371Atrib.

**Comment by CarstenHoffmannMarburg** [comment]
> Malij: https://atlas.paths-erc.eu/places/383

I don't understand, why you are certain, that this place is an ancient place called Malij. In the record, the site name is Deir el-Malek Mikhail (Qibli Qamula). I don't see any reference to a place with the name Malij or any connection. In wikipedia, I find another place with this name in Lower Egypt: https://en.wikipedia.org/wiki/Milig,_Egypt
Are you mistaken or is there another source, that points to Upper Egypt?

**Comment by eu-genia** [comment]
Yes sorry this  is Malij https://ccdl.claremont.edu/digital/collection/cce/id/1251/

**Comment by CarstenHoffmannMarburg** [comment]
So I will encode Michael as a bishop of Atripe and Malig - both in the Delta.
